docs/source/markdown/podman-create.1.md

```diff
@@ -868,13 +878,13 @@ Security Options
 - `seccomp=unconfined` : Turn off seccomp confinement for the container
 - `seccomp=profile.json` :  White listed syscalls seccomp Json file to be used as a seccomp filter
 
+- `proc-opts=OPTIONS` : Comma separated list of options to use for the /proc mount. More details for the
+  possible mount options are specified at **proc(5)** man page.
```

Comment:
Language: seems jarring. I know these two are moved from below, but if you have the chance to fix it, maybe it would read better as "...in \*\*proc(5)\*\*" or "...in the \*\*proc(5)** man page."

Reply:
fixed